File: fyc-api-rest/src/main/java/fr/esgi/fyc_api_rest/dto/response/SeanceResponseDTO.java
package fr.esgi.fyc_api_rest.dto.response;

import java.util.Date;
import java.util.List;

public class SeanceResponseDTO {
    public Long id;
    public String libelle;
    public Date date;
    public int objectifCalorique;
    public SportifResponseDTO sportifResponse;
    public List<ExerciceResponseDTO> exercicesResponse;
}

File: fyc-rest-api-csharp-test/CometeAPI/DTO/Response/SeanceResponseDTO.cs
namespace FycRestApi.DTO.Response;

public class SeanceResponseDTO
{
    public int id { get; set; }
    public string libelle { get; set; }
    public DateTime date { get; set; }
    public int objectifCalorique { get; set; }
    public SportifResponseDTO sportifResponse { get; set; }
    public List<ExerciceResponseDTO> exercicesResponse { get; set; }
}
